src/js/ctlevent.js

```diff
@@ -128,19 +128,22 @@ CTLEvent.prototype.render = function() {
 
     returnString += '</h3><h4>';
 
+    var endDateTimeString = this.endDate.toLocaleTimeString('en-US', timeOptions);
+    var startDateTimeString = this.startDate.toLocaleTimeString('en-US', timeOptions);
+    var endDateString = this.endDate.toLocaleString('en-US', options);
+    var startDateString = this.startDate.toLocaleString('en-US', options);
+
     if (this.multiDay) {
-        returnString += this.startDate.toLocaleString('en-US', options) + ' ' +
-            this.startDate.toLocaleTimeString('en-US', timeOptions) + '<br/>' +
-            'to ' + this.endDate.toLocaleString('en-US', options) + ' ' +
-            this.endDate.toLocaleTimeString('en-US', timeOptions);
+        returnString += startDateString + ' ' + startDateTimeString + '<br/>' +
+            'to ' + endDateString + ' ' + endDateTimeString;
+    } else if (startDateTimeString === endDateTimeString) {
```

Comment:
Should the startDateString & endDateString be compared here as well? Or, do we trust that `multiDay` is taking care of date ranges?

Reply:
Hi Susan, I believe they are being compare on line 143. Though looking at the code, these can be merged together with an `||` since they return the same string. Would that be better?